### PR TITLE
test: adds csv e2e tests

### LINKED ISSUE #546: Add CSV uploader e2e tests
Add e2e tests for CSV uploader. Make sure to handle edge cases such as:
- multi-measurement with different _value types
- incorrectly formatted CSV to test error condition
- _value type of string with special characters such as space and comma

### PR DESCRIPTION
**Pull request by genehynson**
Closes #546, #604

Adds basic e2e tests for CSV uploader:

- uploads a properly formatted csv, checks for success
- uploads an improperly formatted csv, checks for failure

Also fixes issue where the Bucket Settings popup wouldn't work when the csvUploader flag was enabled.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 49 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

File: cypress/e2e/shared/buckets.test.ts [hunk 1]
```diff
@@ -416,4 +420,73 @@ describe('Buckets', () => {
...
+          cy.get('@bucket').then(({id: bucketID}: Bucket) => {
+            cy.visit(`${orgs}/${orgID}${buckets}/${bucketID}/csv-uploader/new`)
+            cy.location('pathname').should(
+              'contain',
+              `${orgs}/${orgID}${buckets}/${bucketID}/csv-uploader/new`
+            )
+          })
+
+          // wait for animation to complete
+          cy.wait(2000)
```
Comment (joelfried) [inline cypress/e2e/shared/buckets.test.ts:438]: Is there anything visible we can interact with instead of a hard coded 2s wait?  It looks like if there's something we could have cypress `.get` that is animating the get function will wait for the animation to complete.

**Review by joelfried — APPROVED** [review]

File: cypress/e2e/shared/buckets.test.ts (hunk 1 above)
Comment (genehynson) [reply cypress/e2e/shared/buckets.test.ts:438]: I tried `.get`ing on a couple elements but couldn't get cypress to wait long enough for the component to fully render lol. It kept dragging and dropping the csv before the popup was even visible 😆  Oh you know I did have this idea yesterday that maybe it's because I'm navigating via the URL rather to the uploader rather than clicking the "add data" button. Let me try that approach and see if it helps at all

Comment (genehynson) [reply cypress/e2e/shared/buckets.test.ts]: woo got it to work!
